src/compute-client/src/command.rs

```diff
@@ -44,7 +44,9 @@ include!(concat!(env!("OUT_DIR"), "/mz_compute_client.command.rs"));
 /// Commands related to the computation and maintenance of views.
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
 pub enum ComputeCommand<T = mz_repr::Timestamp> {
-    /// Indicates the creation of an instance, and is the first command for its compute instance.
+    /// Create the timely runtime according to CommunicationConfig
+    CreateTimely(CommunicationConfig),
```

Comment:
Can you explain in a comment (here or in some other place where the communication protocol is described) why we need the separate command, and why it cannot be bundled together with `CreateInstance`?

Reply:
I improved the comments